### DISCUSSION TITLE
[Bug]: Swedish characters show as upside down questionmark

### DISCUSSION THREAD

**Discussion by Ztrby** [meshtastic/firmware#4932] [Troubleshooting]
### Category

Hardware Compatibility

### Hardware

Heltec V3

### Firmware Version

2.50 and up

### Description

Swedish characters are replaced by upside down questionmarks in the display of all my devices both with e-inc and normal displays.   ->  åäö

### Relevant log output

```Shell
in the app i looks all good, so it have to be a display issue
```

**Comment by Ztrby** [comment]
maybe have somthing to do with issue #4572

**Comment by thebentern** [comment]
You need a regional font like that.

**Reply by Ztrby** [reply]
its part of the ascii 128-255

å = 134 dec or 86 hex
ä = 132 dec or 84 hex
ö = 148 dec or 94 hex

Å = 143 dec or 8F hex
Ä = 142 dec or 8E hex
Ö = 153 dec or 99 hex

**Comment by caveman99** [comment]
The device memory is very VERY limited, so we only have the basic ASCII charset in there. Custom maps can be added. At the moment we have RU, UA and PL as custom charsets.

**Reply by Ztrby** [reply]
so are there only the first 127 characters?

**Reply by garthvh** [reply]
Yes a custom map would need to be added for Swedish
